beku.nix

```diff
@@ -1,11 +1,19 @@
-{ python3 }:
-with python3;
+{ python3, runCommand, lib }:
 let
-  manifest = (pkgs.lib.importTOML ./pyproject.toml).project;
+  # I wish there was a better way. If the command fails, it is very hard to see 
+  # that the problem is here. Perhaps there is some way to evaluate it in python
+  # and reference __version__ directly?
+  version = builtins.readFile (runCommand "foo" { src = [ ./src ]; } ''
+    grep -E '^__version__' $src/beku/version.py | grep -o '".*"' | tr -d \" > $out
```

Comment:
Another less fragile way to do this would be `PYTHONPATH=$src ${python3}/bin/python -c 'import beku.version; print(beku.version.__version__)'`.

Reply:
This is basically what I wanted, just couldn't think how to do it (get the python bin).
Will do that.

Reply:
Fixed in 9a76ed3
Tested by changing the version and seeing it appear in the nix-build output